### PR TITLE
use the terminal color scheme for the scrollbar

### PR DESCRIPTION
**Pull request by Mel34**
https://phabricator.kde.org/D11184

### CODE REVIEW

**Comment by worldofpeace** [comment]
Hi @Mel34,
I'll merge this as soon as I test that it build's :)

File: konsole/PKGBUILD [hunk 1]
```diff
@@ -58,7 +59,8 @@ sha256sums=('fa0997c14a96252177e4808636ec43509eadb1482d54bb8a86b85810283f6cbc'
             'ee2217c8f4a49fc8c0f0f249daa601a3b05f6d7e2f0f7c26c0071bd975bf7358'
             'b6cb7b6d072396d44de0f097a27ae1546602730505c869f1d922c54d8df5f66a'
             '3f9f5f0c0d3df09432b5ccf80496c07069f9d78fbaf262d69ed6cfdc9a158fe6'
-            'af52aa81411395f7aec9621ff512d69d5cd9faad4ab9bf1f238f42fad8dd007d')
+            'af52aa81411395f7aec9621ff512d69d5cd9faad4ab9bf1f238f42fad8dd007d'
+            '0ef7aafaebbee42387a207fcacb711da61644f84285cfb160002679116996c7d')
```
Comment (worldofpeace) [inline konsole/PKGBUILD:63]: The hash here isn't valid. If you run `makepkg -g` It'll generate the hashes for you and you can just copy paste.

File: konsole/PKGBUILD [hunk 2]
```diff
@@ -123,7 +125,9 @@ prepare() {
     patch -p1 -i ../kdebug-328287.patch
   
   msg "kdebug 386264 patch" # See: https://bugs.kde.org/show_bug.cgi?id-386264 Fixed in 18.04.0
-    patch -p1 -i ../kdebug-386264.patch   
+    patch -p1 -i ../kdebug-386264.patch 
+  msg "scrollbars patch" # See: https://phabricator.kde.org/D11184
```
Comment (worldofpeace) [inline konsole/PKGBUILD:129]: A newline before here and we are good.

**Review by worldofpeace — CHANGES_REQUESTED** [review]

**Review by Mel34 — COMMENTED** [review]
I generated sha256 from the local patch file, it probably had an extra line over the git pull because of the way firefox handles copy.

**Comment by worldofpeace** [comment]
@Mel34 Merged, Thank you :+1:

**Pull request merged**